README.md

```diff
@@ -49,8 +49,7 @@ Because users often have their own preferences for which variant of Tensorflow t
 ### Installation:
 - Install ONNX master branch from source. 
 - Install Tensorflow >= 2.2.0 and tensorflow-addons. (Note for Tensorflow 1.x please refer the [tf-1.x branch](https://github.com/onnx/onnx-tensorflow/tree/tf-1.x))
-- Run `git clone https://github.com/onnx/onnx-tensorflow.git && cd onnx-tensorflow`.
-- Run `pip install -e .`.
+- Run `pip install git+https://github.com/onnx/onnx-tensorflow.git`.
```

Comment:
This is an alternative way to install without clone. The current instruction is valid for onnx-tf development since the source code will be pulled to local system. This line is in the "Development" section. So I wonder if you can keep the original instruction and add the new one as a second approach without cloning the repo?